test/jdk/java/nio/file/Files/probeContentType/Basic.java

```diff
@@ -110,8 +110,9 @@ static int checkContentTypes(ExType[] exTypes)
                             + " cannot be determined");
                     failures++;
                 } else if (!expectedTypes.contains(type)) {
-                    System.err.printf("Content type: %s; expected: %s%n",
-                        type, expectedTypes);
+                    System.err.printf("For extension " + extension
+                            + " we got content type: %s; expected: %s%n",
+                            type, expectedTypes);
```

Comment:
For consistency, we can pass `extension` as argument, too:
```
System.err.printf("For extension %s we got content type: %s; expected: %s%n", 
        extension, type, expectedTypes);
```

Reply:
Thanks, I adjusted the output.